docs/api/index.md

```diff
@@ -244,6 +245,62 @@ test('props', () => {
 })
 ```
 
+### vModel
+
+Similar to `props`, but values passed to `vModel` will be updated when using the [`update:modelValue` syntax](https://v3.vuejs.org/guide/component-custom-events.html#v-model-arguments).
+
+**Signature:**
+
+```ts
+vModel?: (RawProps & Props) | ({} extends Props ? null : never)
+```
+
+**Details:**
+
+`Component.vue`:
+
+```vue
+<template>
+  <input @input="updateValue" />
+  {{ color }}
+</template>
+
+<script>
+export default {
+  props: {
+    color: String
+  },
+  methods: {
+    handle($event) {
+      this.$emit('update:color', $event.target.value)
+    }
+  }
+}
+</script>
+```
+
+`Component.spec.js`:
+
+```js
+it('updates v-model automatically', async () => {
+  const wrapper = mount(Component, {
+    vModel: {
+      color: 'red'
+    }
+  })
+
+  expect(wrapper.html()).toContain('red')
+
+  // update the input value
+  wrapper.find('input').element.value = 'blue'
+  // trigger the input event
+  await wrapper.find('input').trigger('input')
```

Comment:
I might be missing something here, but I would've expected something along the lines of:

```suggestion
  await wrapper.find('input').setValue('blue')
```